Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -6,7 +6,7 @@ const personOne = {
 
 // Update the parameter to this function to make it work.
 // Don't change anything else.
-function introduceYourself(___________________________) {
+function introduceYourself({name, age, favouriteFood}) {
```

Comment:
Good work, are there any other solutions you can think of?

Reply:
well, we can loop through keys or just do destructing inside